sdcm/nemesis.py

```diff
@@ -1056,6 +1056,35 @@ def disrupt_modify_table(self):
         disrupt_func = getattr(self, disrupt_func_name)
         disrupt_func()
 
+    def disrupt_mgmt_backup(self):
+        self._set_current_disruption('ManagementBackup')
+        if not self.cluster.params.get('use_mgmt', default=None):
+            raise UnsupportedNemesis('Scylla-manager configuration is not defined!')
+        if not self.cluster.params.get('backup_bucket_location'):
+            raise UnsupportedNemesis('backup bucket location configuration is not defined!')
+
+        manager_node = self.monitoring_set.nodes[0]
+        manager_tool = mgmt.get_scylla_manager_tool(manager_node=manager_node)
+        self.log.debug("sctool version is : {}".format(manager_tool.version))
+
+        cluster_name = self.cluster.name
+        mgr_cluster = manager_tool.get_cluster(cluster_name)
+        if not mgr_cluster:
+            self.log.debug("Could not find cluster : {} on Manager. Adding it to Manager".format(cluster_name))
+            targets = [[n, n.ip_address] for n in self.cluster.nodes]
+            mgr_cluster = manager_tool.add_cluster(name=cluster_name, host=targets[0][1], disable_automatic_repair=True,
+                                                   auth_token=self.monitoring_set.mgmt_auth_token)
+        bucket_location_name = self.cluster.params.get('backup_bucket_location').split()
+        region = self.cluster.params.get('region_name').split()
+        for node in self.cluster.nodes:
+            update_config_file(node=node, region=region[0])
+        mgr_task = mgr_cluster.create_backup_task({'location': ['s3:{}'.format(bucket_location_name[0])]})
```

Comment:
What do we create a backup task for?
All keyspaces? All tables?
Wouldn't we want to add something randomly picked? (not sure, thinking out loud)

Reply:
At this point we do backup the whole database, and it will allow us, in different scenarios, to test different schemas and data sets. We could add a selector, that once time will pick a single KS, other time a single table, or other time the whole DB....

Reply:
How long does it take now to backup everything?
For example, the 1TB longevity.

Reply:
I haven't tested with 1TB data set, but the 100GB data set took average of 30 minutes

Reply:
please start a job in 3.3 reproducers.
1TB job for 3 days, only backup nemesis.
I would like to know how long it takes.